app/themes/3column/ui/styles/theme.less

```diff
@@ -0,0 +1,533 @@
+@primary: red;
+@secondary: black;
+
+html { height: 100% }
+
+body {
+  height: 100%;
+  background-color: #f2f2f2;
+  padding-top: 105px;
+  font-family: "Lucida Sans","Lucida Grande","Lucida Sans Unicode","LucidaSan-Rom",Verdana,sans-serif;
+}
+
+.navbar-static-top {
+  margin-bottom:20px;
+}
+
+i {
+  font-size:16px;
+}
+
+.nav > li > a {
+  color:#787878;
+}
+
+footer {
+  margin-top:20px;
+  padding-top:20px;
+  padding-bottom:20px;
+  background-color:#efefef;
+}
+
+/* count indicator near icons */
+.nav>li .count {
+  position: absolute;
+  bottom: 12px;
+  right: 6px;
+  font-size: 9px;
+  background: rgba(51,200,51,0.55);
+  color: rgba(255,255,255,0.9);
+  line-height: 1em;
+  padding: 2px 4px;
+  -webkit-border-radius: 10px;
+  -moz-border-radius: 10px;
+  -ms-border-radius: 10px;
+  -o-border-radius: 10px;
+  border-radius: 10px;
+}
+
+.btn-primary {
+  background-color: @secondary;
+}
+
+.glyphicon {
+  font-size: 1em;
+  color: @primary;
+}
+
+.container-fluid { min-height: 93%; }
+
+.navbar {
+  border-top: solid 26px @secondary;
+  height: 90px;
+  border-bottom: solid 1px #D2D2D2;
+  margin-bottom: 0;
+
+  &.navbar-default {
+    background-color: #FFF;
+  }
+
+  .navbar-brand {
+    background-image: url(/images/marklogic.png);
+    background-repeat: no-repeat;
+    background-position-y: 0px;
+    background-position-x: 0px;
+    background-size: 200px auto;
+    margin: 10px 0 0 16px;
+    width: 215px;
+  }
+
+  .navbar-form .form-control {
+    width: 325px;
+  }
+
+  .ml-search {
+    padding-right: 5px;
+    margin-top: 4px;
+
+    .form-control[type="text"] {
+      width: 525px;
+    }
+
+    .btn[type="submit"] {
+      background-color: @primary;
+      .glyphicon { color: #fff; }
+    }
+  }
+
+  .user {
+    position: absolute;
+    top: -22px;
```

Comment:
My navbar user/logout button didn't seem properly aligned and was in the white area instead of the black, it seemed like `top: -38px` might be more accurate?